### DISCUSSION TITLE
.GetOrders returns only 6 orders regardless of date

### DISCUSSION THREAD

**Discussion by wheelersf** [abuzuhri/Amazon-SP-API-CSharp#78] [General]
Hi,

No matter what I change the date to, the following request will return no more than 6 orders. The behavior is the same whether I am using .CreatedAfter or .LastUpdateAfter parameters.  The 6 orders returned are also the most recent order instead of starting with the oldest.   Am I missing some configuration in the request, or, could there be something constraining this in code?

Thanks,
Scott

```
//Get Orders List
ParameterOrderList serachOrderList = new ParameterOrderList();
serachOrderList.CreatedAfter = DateTime.UtcNow.AddHours(-240);
serachOrderList.OrderStatuses = new List<OrderStatuses>();
serachOrderList.OrderStatuses.Add(OrderStatuses.Unshipped);
serachOrderList.MarketplaceIds = new List<string> { MarketPlace.US.ID };
var orders = amazonConnection.Orders.GetOrders(serachOrderList);
```

**Comment by abuzuhri** [comment]
Hello @wheelersf

I don't think there any error here its work perfect from my side and and i can take all orders without problem .

can remove the status and add more days instead of hours ?

**Comment by wheelersf** [comment]
Thanks @abuzuhri .  Stupid me.  Not sure how I missed that.  Please accept my apologies. 

Scott

**Reply by abuzuhri** [reply]
What was the problem?

**Reply by wheelersf** [reply]
My hard-coded reference to status that you pointed out : )

Sorry, and thanks again.
